Landing/app/page.tsx

```diff
@@ -77,14 +80,14 @@ export default function Home() {
             <Header />
             {burger && <MobileMenu />}
             {!burger && <>
-              <FirstSection />
```

Comment:
const sections = [
  FirstSection,
  SecondSection,
  ThirdSection,
  FourthSection,
  FifthSection,
  SixthSection,
  SeventhSection,
  EighthSection
];
we can do something like this
{!burger && sections.map((SectionComponent, index) => (
  <FadeInSection key={index}>
    <SectionComponent />
  </FadeInSection>
))}

Reply:
done